test.py

```diff
@@ -317,7 +407,11 @@ async def reap(done, pending, signaled):
         return failed
 
     await start_3rd_party_services(tempdir_base=pathlib.Path(options.tmpdir), toxiproxy_byte_limit=options.byte_limit)
-
+    total_tests = 0
+    for i in range(1, options.repeat+1):
+        result = run_pytest(options, run_id=i)
```

Comment:
Do I get it right, that this runs pytest 3 times sequentially?

IIRC today's --repeat option _queues_ the necessary amount of repetitions and may run them in parallel. Switching it to sequential invocation can be unwanted surprise -- nowadays running a single test with `--repeat $large_number` can be used to stress a test in an attempt to trigger some rare internal race. Sequential invocation no longer supports it.